src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -39,17 +27,21 @@ export interface HelperErrorTextProps {
  * This pattern guarantees accessibility guidelines are met if the text content
  * is dynamically updated by the app or component that implements it.
  */
+
+type ExtendedHelperErrorTextProps = HelperErrorTextProps &
```

Comment:
Three comments:
- you can `extends` the interface with the second clause (interfaces are not preferable over types, but we've been using them throughout the repo, so just for consistency)
- export the prop so consuming apps can use it if needed,
- I'd rename `HelperErrorTextProps` to something more generic like `BaseHelperErrorTextProps` and then call the new one `HelperErrorTextProps`. When this is exported, it'll make more sense for a consumer to use `HelperErrorTextProps` rather than `ExtendedHelperErrorTextProps`.